appveyor.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
```

Comment:
Why should this be a bash/sh file, makes it more complex to reproduce, and there are already batch and powershell. Also this script does very little, so it hardly should a problem to reproduce it in PowerShell.

Reply:
Need to use bash in order to run the tests for dmd, so there's no avoiding it completely. Can make it all a powershell script except for the part that's needed to run the tests.

Reply:
I agree, it's not a big deal to use bash here. It is not an additional dependency as the tests require it anyway.